spec/grape/dsl/configuration_spec.rb

```diff
@@ -70,6 +70,16 @@ class Dummy
           expect(subject.namespace_setting(:description)).to eq(expected_options)
           expect(subject.route_setting(:description)).to eq(expected_options)
         end
+
+        it 'can be set with options and a block' do
+          expect(subject).to receive(:warn)
+
+          desc_text = 'The description'
+          options = { message: 'none' }
+          subject.desc desc_text, options do
+            detail 'more details'
+          end
```

Comment:
It would be great to ensure explicitly that the `options` hash is *not* merged here.